docs/source/configuration/logging.mdx

```diff
@@ -1,8 +1,29 @@
 ---
-title: Configuring Logging
-description: Tuning logging configurations
+title: Logging
+description: Logging output formats
 ---
 
 import { Link } from 'gatsby';
 
-The Apollo Router automatically uses JSON-formatted logging when there is not an interactive shell session attached (e.g., in CI and deployed environments). JSON-formatted logging facilitates compatibility with searchable logging facilities like Google Cloud Logging or Stackdriver.
\ No newline at end of file
+The Apollo Router automatically uses JSON-formatted logging when there is not an interactive shell session attached (e.g., in CI and deployed environments).
+
+## Basic logging via shell
+
+Basic logging is mostly used during local development.
+
+```bash
+2022-03-18T10:28:11.090054Z  INFO apollo_router::executable: apollo-router@0.1.0-alpha.9
+2022-03-18T10:28:11.160650Z  INFO apollo_router: Starting Apollo Router
+2022-03-18T10:28:11.339906Z  INFO apollo_router: Listening on http://127.0.0.1:4000 🚀
+```
+
+## JSON-formatted logging:
+
+JSON-formatted logging facilitates compatibility with searchable logging facilities like Google Cloud Logging or Stackdriver. It is preferred for production environments.
```

Comment:
How do I change it? Can I change it?  If so, we should add that here, or at least add an explanation of how we choose.

Reply:
You can't change it. It's auto-detected when the router starts.

Reply:
I added a sentence to state that it was the Bunyan format, and that it was popular.